src/models/CheckoutParams.ts

```diff
@@ -0,0 +1,35 @@
+import AmountMoney from "./AmountMoney";
+import TipSettings from "./TipSettings";
+
+interface CheckoutParams {
+    brand?: string;
+
+    expirationMonth?: number;
+
+    expirationYear?: number;
+
+    lastFourDigits?: string;
+
+    postalCode?: string;
+
+    prepaidType?: string;
+
+    type?: string;
+
+    amountMoney?: AmountMoney;
```

Comment:
I believe this should a required field and please put in on top of the list?

Reply:
It's fixed.